src/main/java/uk/gov/digital/ho/hocs/audit/application/ZonedDateTimeConverter.java

```diff
@@ -39,11 +39,12 @@ public ZonedDateTimeConverter(final String outputtedDateFormat, final String tim
      * @return the offsetted date time string
      */
     public String convert(final LocalDateTime localDateTime) {
+        log.info("Timestamp before convert: {}", localDateTime);
```

Comment:
Logger is not valid in this case - you will need to add the annotation to the top of this file to use Lomboks.

Reply:
Build passing now, resolved.